### ISSUE TITLE
Headless UI Combobox?

### ISSUE THREAD

**Issue by KieranFleckney** [DavidVollmers/Ignis#48]
Is the combobox component in the work or is this something you are not looking to implement currently? Was there a reason you didn't do port it with the others?

**Comment by DavidVollmers** [comment]
Hey thanks for reaching out!

So to be honest I didn't implement the ComboBox yet for two reasons:

1. I didn't need to use it myself so far
2. I am scared about the efforts of implementing and maintaining it. It is a rather complex component and also it will be really difficult to deliver the same developer experience as the JS packages of Headless UI do. One part for example which will take some brainstorming to implement is [Binding objects as values](https://headlessui.com/react/combobox#binding-objects-as-values).

So I want to definetly implement it at some point but was lacking the motivation so far... I hope I will find some time in the upcoming weeks. On the other hand there is also a big update on Headless UI in the pipeline which will introduce even more new components which I want to implement using Ignis (https://github.com/tailwindlabs/headlessui/releases/tag/%40headlessui%2Freact%40v2.0.0-alpha.1). But before I will start with this I will make sure to implement the ComboBox!

**Comment by DavidVollmers** [comment]
I will leave this issue open until the ComboxBox component is implemented and I will also pin it for visibility.

**Comment by KieranFleckney** [comment]
@DavidVollmers I appreciate the reply. I thought reason one would likely be the case. Also v2 looks good. Thanks for the great work.

Also would it be a good idea to enable Discussions for the repo. Would be good to ask question about the library that are not issues? Or are you happy them being open as issues?

**Comment by DavidVollmers** [comment]
Good point! I enabled discussions right away. Completely forgot that feature exists...
